### PR TITLE
Fix the compatibility with MPForms v5

### PR DESCRIPTION
**Pull request by qzminski**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,16 +27,16 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
-        "contao/core-bundle": "^4.9",
+        "contao/core-bundle": "^4.9 || ^5.0",
```
[the remark this answers is not in the document]
Comment (Toflar) [reply composer.json:30]: I mean I know that mp_forms is optional but I think if we release a new minor version with some cleanup and make it at least 4.13, that wouldn't hurt either :)

[the remark this answers is not in the document]
Comment (qzminski) [reply composer.json]: I did bump to Contao 4.13 now, but not sure if we should touch PHP really. It does not harm to have it for 7.4, too.

**Comment by Toflar** [comment]
I've released https://github.com/terminal42/contao-mp_forms/releases/tag/5.0.0 so we're good to get this ready as well :)

File: composer.json [hunk 2]
```diff
@@ -27,16 +27,16 @@
...
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
-        "contao/core-bundle": "^4.9",
+        "contao/core-bundle": "^4.13 || ^5.0",
         "doctrine/dbal": "^2.11 || ^3"
     },
     "require-dev": {
         "contao/manager-plugin": "^2.0",
-        "terminal42/contao-mp_forms": "^4.4"
+        "terminal42/contao-mp_forms": "^5.0"
```
Comment (Toflar) [inline composer.json:35]:
```suggestion
        "terminal42/contao-mp_forms": "^4.4 || ^5.0"
```

Otherwise you'll have a hard time installing v4 ;)

**Review by Toflar — CHANGES_REQUESTED** [review]

**Review by Toflar — APPROVED** [review]

**Pull request merged**